src/templates/semantic/wizard/builder.hbs

```diff
@@ -0,0 +1 @@
+<div style="color:#6c757d; text-align:center; padding: 0.5rem">{{ t(component.title) }}</div>
```

Comment:
We generally don't use inline styles since users may need to override them. Is it possible to add these styles in the CSS based on the generated class names?

Reply:
@randallknutson I based this on existing inline css for semantic ui. Do you have a preference for where where the css class definitions should go?

Reply:
@randallknutson Moved the inline css into cssClasses.js but not sure if this is the right place..